app/controllers/proposals_controller.rb

```diff
@@ -74,6 +75,17 @@ def update
     end
   end
 
+  def toogle_favorite
+    user = User.find(params[:favourite_user_id])
+    users = @event.favourite_users
+    if users.include? user
```

Comment:
`if users.include?(user)` ? Because that's not needed, do we prefer it? I don't.